src/test/kotlin/Racing.kt

```diff
@@ -0,0 +1,77 @@
+// import jdk.internal.util.xml.impl.Input
+import kotlin.random.Random
+
+// import org.assertj.core.api.InstanceOfAssertFactories.array
+// import java.lang.NumberFormatException
+
+interface Inputall{                         // 입력 인터페이스
+    fun inputnumber() : Int
+    fun inputcircuit(): Int
+}
+
+
+class InputView : Inputall{                 // 입력에 대한 클래스
```

Comment:
space convention 위반
수업 중에 다룬 lint를 적용하거나 intellij의 formatting 활용해 convention 지키는 습관을 가지면 좋겠다.

단축키는
Opt + Cmd + L (macOS)
Ctrl + Alt + L (other)

Reply:
```suggestion
class InputView : Inputall{                 // 입력에 대한 클래스
```
전에도 언급해 주셨는데 좀 더 신경쓰겠습니다.